README.md

```diff
@@ -9,6 +9,11 @@
 
 # Development tasks
 
+## update packages
+
+- Make sure you have ruby install and that it is available in your path
+- Run `rake postclean` or simply double click on `postclean.bat`. This will update all nuget packages and copy the dependencies in the package `inst/lib` folder.

```

Comment:
@PavelBal Make sure you have ruby install and in your path 
Then simply execute postclean.bat (with R Studio closed) to download all components

Note. you also need to do `git submodule update --init --recursive` do update the submodule

Reply:
We will need a clear step-by-step instruction in the readme before the first release. Downloading and compiling the package should be easy.

Reply:
Why do you expect that people will want to compile the package? We will provide zip as tar package as all other R packages not on CRAN.